src/pages/community/index.astro

```diff
@@ -58,26 +62,21 @@ import Images from "./images.astro";
       </div>
     </Container>
     <Container class:list={"!max-w-5xl"}>
-      <Images />
+      <Images/>
     </Container>
 
-    <Container class:list={"!max-w-5xl"}>
+    <Container class="!justify-center !max-w-5xl">
       <h2 class="text-3xl md:text-5xl text-center my-8 lg:my-16 !max-w-none">
         Join the community
       </h2>
 
-      <p class="my-6">
-        To join our Barcelona chat group, hit the button below!
-      </p>
-
-      <div class="flex mt-6 justify-center">
-        <Link
-          href="https://chat.whatsapp.com/D09VYKJ7x5pGIhdkxp1Eze"
-          style="success"
-          target="_blank"
-        >
-          Join (WhatsApp Group)
-        </Link>
+      <div class="!max-w-xl" style="margin: 0 auto;">
```

Comment:
Would probably work with just this:

<div class="max-w-xl mx-auto my-0">

Those are the tailwind utility class equivalents. Also, don't need the exclamation mark on max width probable